### ISSUE TITLE
Migrate to Wolfi

### ISSUE THREAD

**Issue by erikaheidi** [chainguard-images/php#13]
This image still uses Alpine as base. We should migrate it to use Wolfi. There have been a few blockers to execute this migration, so I will track the progress in this issue.

**Comment by amouat** [comment]
The way we've approached this in other repos is to have a `apko.yaml` _and_ an `apko-glibc.yaml`, with the glibc one being wolfi based, but keeping the alpine one as well. 

In the future, we will probably have a musl version of wolfi, but until then we should keep the alpine version alongside.

**Comment by erikaheidi** [comment]
right, that makes sense! it's also a good starting point since this image is not compiling from source yet. I am looking at this now.

**Comment by amouat** [comment]
@erikaheidi I think you're working on this, so I assigned you. Please ask if you need any help!
